### DISCUSSION TITLE
FAIL

### DISCUSSION THREAD

**Discussion by Thiago-Offredi** [react-hook-form/react-hook-form#10189] [General]
Failed to compile.

Error in ./~/react-hook-form/dist/index.cjs.js
Module parse failed: /home/thiagooffredi/code-burger-interface/node_modules/react-hook-form/dist/index.cjs.js Unexpected token (1:1685)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (1:1685)
 @ ./src/containers/login/index.js 12:21-47

**Comment by leapful** [comment]
@Thiago-Offredi 

Please provide a codesandbox or github repo to re-produce the issue.
